internal/serverinstall/k8s.go

```diff
@@ -1353,6 +1487,27 @@ func (i *K8sInstaller) InstallFlags(set *flag.Set) {
 		Default: defaultServerImage,
 	})
 
+	set.StringVar(&flag.StringVar{
+		Name:    "k8s-odr-image",
+		Target:  &i.config.odrImage,
+		Usage:   "Docker image for the Waypoint On-Demand Runners",
+		Default: defaultODRImage,
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:   "k8s-runner-service-account",
+		Target: &i.config.odrServiceAccount,
+		Usage: "Service account to assign to the on-demand runner. If this is blank, " +
+			"a service account will be created automatically with the correct permissions.",
```

Comment:
nit: could add `Default: "waypoint-runner"` here and omit the `if i.config.odrServiceAccount == ""` check above. Or would that cause problems with the config returned in `OnDemandRunnerConfig()`?